compiler/cpp/src/thrift/generate/t_py_generator.cc

```diff
@@ -105,7 +106,9 @@ class t_py_generator : public t_generator {
         }
         if( import_dynbase_.empty()) {
           import_dynbase_ = "from thrift.protocol.TBase import TBase, TFrozenBase, TExceptionBase, TFrozenExceptionBase, TTransport\n";
-        }
+        } 
+      } else if( iter->first.compare("immutable_exc") == 0) {
```

Comment:
shouldn't this be called `mutable_exc` instead? otherwise you are making `gen_immutable_exc_` to false when user pass in `immutable_exc` arg to the compiler.

Reply:
Yea this naming is a bit confusing, I'll update it :)